System/Console/ANSI/Color.hs

```diff
@@ -2,50 +2,59 @@
 module System.Console.ANSI.Color
   (
   -- | 8-bit to 24-bit color conversion
-    color8To24
+    xterm256ToSRGB
   -- | Utility functions
   , clamp
-  , color8CodeToEither
   , closest4bitsANSIColor
+  , closestXterm256Color
+  , color8CodeToXterm256
+  , colorToCode
+  , xterm256ColorToCode
     ) where
 
 
-import qualified Data.Colour.SRGB.Linear as SRGBLin(rgb, Colour)
-import qualified Data.Colour.SRGB as SRGB (RGB (..), toSRGB, Colour)
+import Data.Colour.SRGB (Colour, RGB (..), sRGB, toSRGB)
 
 import Data.Word (Word8)
 import Data.Colour.Names (black, blue, cyan, green, grey, lime, magenta, maroon,
                           navy, olive, purple, red, silver, teal, white, yellow)
 
 import Data.List (minimumBy, (!!))
+import Data.Maybe (fromJust)
 
 import System.Console.ANSI.Types
 
--- | Returns a color, in the linear sRGB colorspace, whose components are the components
---  of an 8-bit ANSI color.
+-- | Returns a color, in the linear sRGB colorspace, whose components are the
```

Comment:
here if we chose srgb we should remove the "linear" word

Reply:
Agreed.